### ISSUE TITLE
pmd-unit-tests:JUnit4TestShouldUseTestAnnotation - False Positive on @Parameters method

### ISSUE THREAD

**Issue by ArnaudLec** [jborgers/sonar-pmd#46]
Using sonar-pmd 2.6, I'm greeted with a false positive on the rule pmd-unit-tests:JUnit4TestShouldUseTestAnnotation for the following code :
```java
@Parameters(name = "{0}")
public static final Collection<Object[]> testParameters() {
    return NoSqlDirectoryTestMatrix.TEST_CASES;
}
```
If I rename the `@Parameters` method `parameters` then the false positive disappears.
I think that methods annotated with `@Parameters` shouldn't be flagged for missing the `@Test` annotation.

**Comment by jborgers** [comment]
If still relevant, please report at pmd (https://pmd.github.io/)
